src/ofono/provider_ofono.hpp

```diff
@@ -179,6 +201,7 @@ class Bridge : public QObject, public PropertiesSource
     std::pair<QString, QString> network_name_;
     void (Bridge::*set_name_)();
 
+    QString modem_pattern_;
```

Comment:
I changed it to QRegularExpression, which is a bit heavier. As I mentioned above, I hope that this code can be replaced with a better solution.